web/web.go

```diff
@@ -97,7 +97,7 @@ func (h *handler) handleUniqueIdLookup(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *handler) uniqueIdToArn(ctx context.Context, id string) (string, error) {
-	accessPointName := fmt.Sprintf("awsid-%d", rand.Int31())
```

Comment:
UUID is preferred, if it wouldn't interrupt your workflow with a different naming convention.